api/db-cdr/generate-cdr/make-bq-dataset-linking.sh

```diff
@@ -0,0 +1,271 @@
+fi
+
+
+# Check that bq_dataset exists and exit if not
+datasets=$(bq --project=$BQ_PROJECT ls)
+if [ -z "$datasets" ]
+then
+  echo "$BQ_PROJECT.$BQ_DATASET does not exist. Please specify a valid project and dataset."
+  exit 1
+fi
+if [[ $datasets =~ .*$BQ_DATASET.* ]]; then
+  echo "$BQ_PROJECT.$BQ_DATASET exists. Good. Carrying on."
+else
+  echo "$BQ_PROJECT.$BQ_DATASET does not exist. Please specify a valid project and dataset."
+  exit 1
+fi
+
+
+################################################
+# CREATE LINKING TABLE
+################################################
+echo "CREATE TABLE - ds_linking"
+bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+"CREATE OR REPLACE TABLE \`$BQ_PROJECT.$BQ_DATASET.ds_linking\`
+(
+    DENORMALIZED_NAME               STRING,
+    OMOP_SQL                        STRING,
+    JOIN_VALUE                      STRING,
+    DOMAIN                          STRING,
+)"
+
+################################################
+# INSERT DATA
+################################################
+echo "ds_linking - inserting conditions data"
+
+bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+"INSERT INTO \`$BQ_PROJECT.$BQ_DATASET.ds_linking\`
+VALUES
+    ('PERSON_ID', 'c_occurrence.PERSON_ID', 'from \`$BQ_PROJECT.$BQ_DATASET.condition_occurrence\` c_occurrence', 'Condition'),
+    ('CONDITION_CONCEPT_ID', 'c_occurrence.CONDITION_CONCEPT_ID', 'from \`$BQ_PROJECT.$BQ_DATASET.condition_occurrence\` c_occurrence', 'Condition'),
+    ('STANDARD_CONCEPT_NAME', 'c_standard_concept.concept_name as STANDARD_CONCEPT_NAME', 'left join \`$BQ_PROJECT.$BQ_DATASET.concept\` c_standard_concept on c_occurrence.CONDITION_CONCEPT_ID = c_standard_concept.CONCEPT_ID', 'Condition'),
+    ('STANDARD_CONCEPT_CODE', 'c_standard_concept.concept_code as STANDARD_CONCEPT_CODE', 'left join \`$BQ_PROJECT.$BQ_DATASET.concept\` c_standard_concept on c_occurrence.CONDITION_CONCEPT_ID = c_standard_concept.CONCEPT_ID', 'Condition'),
+    ('STANDARD_VOCABULARY', 'c_standard_concept.vocabulary_id as STANDARD_VOCABULARY', 'left join \`$BQ_PROJECT.$BQ_DATASET.concept\` c_standard_concept on c_occurrence.CONDITION_CONCEPT_ID = c_standard_concept.CONCEPT_ID', 'Condition'),
+    ('CONDITION_START_DATETIME', 'CONDITION_START_DATETIME', 'from \`$BQ_PROJECT.$BQ_DATASET.condition_occurrence\` c_occurrence', 'Condition'),
+    ('CONDITION_END_DATETIME', 'CONDITION_END_DATETIME', 'from \`$BQ_PROJECT.$BQ_DATASET.condition_occurrence\` c_occurrence', 'Condition'),
+    ('CONDITION_TYPE_CONCEPT_ID', 'CONDITION_TYPE_CONCEPT_ID', 'from \`$BQ_PROJECT.$BQ_DATASET.condition_occurrence\` c_occurrence', 'Condition'),
+    ('CONDITION_TYPE_CONCEPT_NAME', 'c_type.concept_name as CONDITION_TYPE_CONCEPT_NAME', 'left join \`$BQ_PROJECT.$BQ_DATASET.concept\` c_type on c_occurrence.CONDITION_TYPE_CONCEPT_ID = c_type.CONCEPT_ID', 'Condition'),
+    ('STOP_REASON', 'STOP_REASON', 'from \`$BQ_PROJECT.$BQ_DATASET.condition_occurrence\` c_occurrence', 'Condition'),
```

Comment:
nit: is this missing a c_occurrence. prefix, e.g. 'c.occurrence.STOP_REASON'? It may be valid SQL if the field name is unique among all joined tables, but it will seem clearer / more consistent to include the table prefix everywhere (here and for all other domains too).

Reply:
I can add this. 
It did actually run, so this is valid SQL, but being clearer can't hurt